viz/visualize.py

```diff
@@ -33,15 +35,22 @@ def __init__(self, model, dataset, model_dir=None, save_images=True):
 
         dataset : str
             Name of the dataset.
+
+        loss_of_interest : str
+            The loss type (as saved in the log file) to order the latent dimensions by and display (optionally)
+
+        display_loss_per_dim : bool
```

Comment:
This should be is_display_loss_per_dim under the Yann convention for good code writing.